doc/api/next_api_changes/removals/22107-OG.rst

```diff
@@ -0,0 +1,44 @@
+Removal of deprecated ``mathtext`` APIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following :mod:`.mathtext` APIs have been removed:
+``MathtextBackendBitmap``,
+``MathtextBackendPs``,
+``MathtextBackendPdf``,
+``MathtextBackendSvg``,
+``MathtextBackendCairo``,
+``ship``,
+``Fonts``,
+``TruetypeFonts``,
+``StandardPsFonts``,
+``FontConstantsBase``,
+``ComputerModernFontConstants``,
+``STIXFontConstants``,
+``STIXSansFontConstants``,
+``DejaVuSerifFontConstants``,
+``DejaVuSansFontConstants``,
+``FontConstantsBase``,
+``Node``,
+``Box``,
+``Char``,
+``Glue``,
+``Kern``,
```

Comment:
Missing all subclasses of these, `Accent`, `List`, etc.

Reply:
Hmm, I wonder what happened here. I used the deprecation code to print much of that list. So those classes did not import in 3.5.1 (and were not deprecated). 

```
In [1]: from matplotlib.mathtext import Accent
Traceback (most recent call last):

  File "C:\Users\Oscar\AppData\Local\Temp/ipykernel_8/2606447054.py", line 1, in <module>
    from matplotlib.mathtext import Accent

ImportError: cannot import name 'Accent' from 'matplotlib.mathtext' (C:\Users\Oscar\miniconda3\lib\site-packages\matplotlib\mathtext.py)


In [2]: import matplotlib

In [3]: matplotlib.__version__
Out[3]: '3.5.1'
```

I guess that the actual problem was maybe that they were removed without deprecation earlier, but I can for sure add them here.

Reply:
Ah yes, I don't think it was intended to remove them, but it happened and no-one has complained so far. The vaguer 'its subclasses' will do instead of enumerating all of them.